src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs

```diff
@@ -527,6 +528,32 @@ public async Task CreateOrUpdatePullRequestCommentAsync(string pullRequestUrl, s
             await client.CreateThreadAsync(newCommentThread, repoName, id);
         }
 
+        public async Task CreateOrUpdatePullRequestMergeStatusInfoAsync(string pullRequestUrl, IReadOnlyList<MergePolicyEvaluationResult> evaluations)
+        {
+            await CreateOrUpdatePullRequestCommentAsync(pullRequestUrl,
+                    $@"## Auto-Merge Status
+                    This pull request has not been merged because Maestro++ is waiting on the following merge policies.
```

Comment:
Do we really want this giant indent in the comment?  Any leading space in a verbatim string is included verbatim in the value.

Reply:
We can change that, I kept the same values as it was before the implementation of the checks.

Reply:
I wonder if the the markdown renderer just removes the extra spaces.

Reply:
Leading spaces mark the text as a code block, so it will get <pre> rendered in a big gray box.

Reply:
##This has no leading spaces
    but this does.

Reply:
Okay yeah. Initial heading and then code block.

It looks like the original code removed the leading spaces on lines 2+, so I'd keep it that way.

Reply:
I think this still needs to be fixed, but otherwise looks good.